### ISSUE TITLE
你好！我想请问呢精度问题或者参数问题

### ISSUE THREAD

**Issue by xuyaoming123** [winycg/CIRKD#33]
def parse_args():
    parser = argparse.ArgumentParser(description='Semantic Segmentation Training With Pytorch')
    # model and dataset
    parser.add_argument('--model', type=str, default='deeplabv3',
                        help='model name')  
    parser.add_argument('--backbone', type=str, default='resnet18',
                        help='backbone name')
    parser.add_argument('--dataset', type=str, default='citys',
                        help='dataset name')
    parser.add_argument('--data', type=str, default='./dataset/cityscapes/',
                        help='dataset directory')
    parser.add_argument('--crop-size', type=int, default=[512, 512], nargs='+',
                        help='crop image size: [height, width]')
    parser.add_argument('--workers', '-j', type=int, default=0,
                        metavar='N', help='dataloader threads')
    
    # training hyper params
    parser.add_argument('--aux', action='store_true', default=False,
                        help='Auxiliary loss')
    parser.add_argument('--aux-weight', type=float, default=0.4,
                        help='auxiliary loss weight')
    parser.add_argument('--batch-size', type=int, default=8, metavar='N',
                        help='input batch size for training (default: 8)')
    parser.add_argument('--start_epoch', type=int, default=0,
                        metavar='N', help='start epochs (default:0)')
    parser.add_argument('--max-iterations', type=int, default=40000, metavar='N',
                        help='number of epochs to train (default: 50)')
    parser.add_argument('--lr', type=float, default=0.02, metavar='LR',
                        help='learning rate (default: 1e-4)')
    parser.add_argument('--momentum', type=float, default=0.9, metavar='M',
                        help='momentum (default: 0.9)')
    parser.add_argument('--weight-decay', type=float, default=1e-4, metavar='M',
                        help='w-decay (default: 5e-4)')
    parser.add_argument('--ignore-label', type=int, default=-1, metavar='N',
                        help='input batch size for training (default: 8)')
    
    # cuda setting
    parser.add_argument('--no-cuda', action='store_true', default=False,
                        help='disables CUDA training')
    parser.add_argument('--local_rank', type=int, default=0)
    # checkpoint and log
    parser.add_argument('--resume', type=str, default=None,
                        help='put the path to resuming file if needed')
    parser.add_argument('--save-dir', default='./save_model',
                        help='Directory for saving checkpoint models')
    parser.add_argument('--save-epoch', type=int, default=10,
                        help='save model every checkpoint-epoch')
    parser.add_argument('--log-dir', default='./runs/logs/',
                        help='Directory for saving checkpoint models')
    parser.add_argument('--log-iter', type=int, default=250,
                        help='print log every log-iter')
    parser.add_argument('--save-per-iters', type=int, default=800,
                        help='per iters to save')
    parser.add_argument('--val-per-iters', type=int, default=800,
                        help='per iters to val')
    parser.add_argument('--pretrained-base', type=str, default='resnet18-imagenet.pth',
                        help='pretrained backbone')
    parser.add_argument('--pretrained', type=str, default='None',
                        help='pretrained seg model')
这是我的训练参数，训练结果却是一直没有变化
2025-07-21 09:42:07,121 semantic_segmentation INFO: Validation pixAcc: 0.985, mIoU: 0.263
2025-07-21 09:46:34,766 semantic_segmentation INFO: Iters: 32250/40000 || Lr: 0.004567 || Task Loss: 0.0459 || Cost Time: 11:01:17 || Estimated Time: 2:38:54
2025-07-21 09:51:01,558 semantic_segmentation INFO: Iters: 32500/40000 || Lr: 0.004434 || Task Loss: 0.0538 || Cost Time: 11:05:44 || Estimated Time: 2:33:37
2025-07-21 09:55:34,011 semantic_segmentation INFO: Iters: 32750/40000 || Lr: 0.004301 || Task Loss: 0.0417 || Cost Time: 11:10:16 || Estimated Time: 2:28:22
2025-07-21 09:56:29,528 semantic_segmentation INFO: Start validation, Total sample: 500
2025-07-21 09:58:28,688 semantic_segmentation INFO: Validation pixAcc: 0.985, mIoU: 0.265
2025-07-21 10:02:03,518 semantic_segmentation INFO: Iters: 33000/40000 || Lr: 0.004167 || Task Loss: 0.0810 || Cost Time: 11:16:46 || Estimated Time: 2:23:33
2025-07-21 10:06:33,265 semantic_segmentation INFO: Iters: 33250/40000 || Lr: 0.004033 || Task Loss: 0.0467 || Cost Time: 11:21:16 || Estimated Time: 2:18:18
2025-07-21 10:11:05,699 semantic_segmentation INFO: Iters: 33500/40000 || Lr: 0.003898 || Task Loss: 0.0715 || Cost Time: 11:25:48 || Estimated Time: 2:13:04
2025-07-21 10:12:55,331 semantic_segmentation INFO: Start validation, Total sample: 500
2025-07-21 10:14:54,876 semantic_segmentation INFO: Validation pixAcc: 0.985, mIoU: 0.264
2025-07-21 10:17:37,027 semantic_segmentation INFO: Iters: 33750/40000 || Lr: 0.003763 || Task Loss: 0.1307 || Cost Time: 11:32:19 || Estimated Time: 2:08:12
2025-07-21 10:22:06,475 semantic_segmentation INFO: Iters: 34000/40000 || Lr: 0.003627 || Task Loss: 0.0764 || Cost Time: 11:36:49 || Estimated Time: 2:02:58
2025-07-21 10:26:33,783 semantic_segmentation INFO: Iters: 34250/40000 || Lr: 0.003491 || Task Loss: 0.0698 || Cost Time: 11:41:16 || Estimated Time: 1:57:43
2025-07-21 10:29:16,681 semantic_segmentation INFO: Start validation, Total sample: 500
2025-07-21 10:31:16,344 semantic_segmentation INFO: Validation pixAcc: 0.985, mIoU: 0.263
2025-07-21 10:33:04,559 semantic_segmentation INFO: Iters: 34500/40000 || Lr: 0.003354 || Task Loss: 0.1044 || Cost Time: 11:47:47 || Estimated Time: 1:52:50
精度一直卡在0.26左右  请问是我哪设置错了吗
